documentation/1.2/tour/inheritance.md

```diff
@@ -301,8 +301,8 @@ an abstract superclass:
 Ceylon requires us to annotate abstract classes `abstract`, just like Java. 
 This annotation specifies that a class cannot be instantiated, and can define 
 abstract members. Like Java, Ceylon also requires us to annotate "abstract" 
-members that don't specify an implementation. However, in this case, the 
-required annotation is `formal`. The reason for having two different 
+members (i.e. members that don't specify an implementation. However, in this
+case, the required annotation is `formal`. The reason for having two different 
```

Comment:
I am not a fan of stuff like "(i.e. blah blah blah)". To me that's just a really lazy writing style.